test/ComposerRequireCheckerTest/FileLocator/LocateComposerPackageSourceFilesTest.php

```diff
@@ -148,6 +152,7 @@ public function testFromPsr4WithNestedDirectoryAlternativeDirectorySeparator():
      *
      * @dataProvider provideExcludePattern
      */
+    #[DataProvider('provideExcludePattern')]
```

Comment:
Do we need both of these comments? Can we remove the docblock tag in favour of the attribute?

Reply:
https://github.com/maglnet/ComposerRequireChecker/pull/564#discussion_r1946739239